test/classes/vass/nested-class-with-user-defined-constructor-1.bad

```diff
@@ -1 +1,9 @@
 nested-class-with-user-defined-constructor-1.chpl:4: error: constructor for class 'inner' requires a generic argument called 'outer'
+nested-class-with-user-defined-constructor-1.chpl:4: In initializer 'inner':
+nested-class-with-user-defined-constructor-1.chpl:4: internal error: EXPnnnn chpl Version mmmm
+
+Internal errors indicate a bug in the Chapel compiler ("It's us, not you"),
+and we're sorry for the hassle.  We would appreciate your reporting this bug -- 
+please see http://chapel.cray.com/bugs.html for instructions.  In the meantime,
+the filename + line number above may be useful in working around the issue.
```

Comment:
I confirmed that the internal error was there before my changes. It is introduced because I added --verify. In our nightly testing this test was tested without --verify.